utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -387,11 +391,21 @@ where
 	};
 
 	let ext = {
-		let builder = Builder::<Block>::new().mode(mode);
-		// We do not allow overwriting code because we need exact state for storage root match.
-		let mut ext = builder.build().await?;
+		let builder = Builder::<Block>::new()
+			.mode(mode)
+			.inject_hashed_key(&[twox_128(b"System"), twox_128(b"LastRuntimeUpgrade")].concat());
+		let mut ext = if command.overwrite_code {
+			let (code_key, code) = extract_code(config.chain_spec)?;
+			builder.inject_key_value(&[(code_key, code)]).build().await?
+		} else {
+			builder
+				.inject_hashed_key(well_known_keys::CODE)
```

Comment:
this is fine, but i don't 100% understand the case when you would not fetch state from the all the runtime modules?

Reply:
@kianenigma, perhaps obvious I also think its worth noting that this will have no affect on the builder if we are using `State::Snap`

Reply:
> this is fine, but i don't 100% understand the case when you would not fetch state from the all the runtime modules?

For example of the offchain worker, you are only interested in executing the OCW of one pallet. Albeit, you must be sure that the OCW of all other pallets won't panic or sth if they don't have any data set in their pallet.

Reply:
@kianenigma but does execute block run the offchain worker?